From c0f489f25d01d205eba41819d6c29131e01ac2e9 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Mon, 19 Nov 2012 17:00:05 +0200
Subject: [PORT FROM MAIN] Telephony: Adding HSPAP (HSPA+) support in Android

BZ: 41786

HSPA+ management is missing in Android framework.

The value NETWORK_TYPE_HSPAP should be added in
switch cases handling network type.

Change-Id: If2f0e6849c1a2ebbf59c4676e37caa17277d2341
Orig-Change-Id: I51cc2bbcc817195380e7947ea43c7f0f279d55df
Category: aosp improvement
Domain: Telephony-Telephony-FWK
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Regnier, PhilippeX <philippex.regnier@intel.com>
---
 src/java/com/android/internal/telephony/RIL.java |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/java/com/android/internal/telephony/RIL.java b/src/java/com/android/internal/telephony/RIL.java
index 824e191..a227008 100644
--- a/src/java/com/android/internal/telephony/RIL.java
+++ b/src/java/com/android/internal/telephony/RIL.java
@@ -24,6 +24,7 @@ import static android.telephony.TelephonyManager.NETWORK_TYPE_UMTS;
 import static android.telephony.TelephonyManager.NETWORK_TYPE_HSDPA;
 import static android.telephony.TelephonyManager.NETWORK_TYPE_HSUPA;
 import static android.telephony.TelephonyManager.NETWORK_TYPE_HSPA;
+import static android.telephony.TelephonyManager.NETWORK_TYPE_HSPAP;
 
 import android.content.BroadcastReceiver;
 import android.content.Context;
@@ -3404,6 +3405,8 @@ public final class RIL extends BaseCommands implements CommandsInterface {
            radioType = NETWORK_TYPE_HSUPA;
        } else if (radioString.equals("HSPA")) {
            radioType = NETWORK_TYPE_HSPA;
+       } else if (radioString.equals("HSPAP")) {
+           radioType = NETWORK_TYPE_HSPAP;
        } else {
            radioType = NETWORK_TYPE_UNKNOWN;
        }
-- 
1.7.4.1

